drivers/node/aws/errors.go

```diff
@@ -0,0 +1,16 @@
+package aws
```

Comment:
Merge `ssh/errors.go` and `aws/errors.go` and move it to `node/errors.go`.

This will remove redundant error objects.

Reply:
merged